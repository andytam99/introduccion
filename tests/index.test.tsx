import { render, screen } from "@testing-library/react";
import Index from "../pages/index";

describe("Index", () => {
	it("Renders Hello", () => {
		render(<Index />);

		const heading = screen.getByRole("heading", {
			name: /Hello/i,
		});

		expect(heading).toBeInTheDocument();
	});
	it("Matches snaptchot", () => {
		const { container } = render(<Index />);
		expect(container).toMatchSnapshot();
	});
});
