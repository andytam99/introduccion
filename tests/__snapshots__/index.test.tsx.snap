// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Matches snaptchot 1`] = `
<div>
  <div
    class="flex w-full justify-center items-center h-full"
  >
    <h1>
      Hello
    </h1>
  </div>
</div>
`;
